lib/acme/client/resources/challenges/dns01.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 class Acme::Client::Resources::Challenges::DNS01 < Acme::Client::Resources::Challenges::Base
   CHALLENGE_TYPE = 'dns-01'.freeze
```

Comment:
Do you still need the `freeze` on top of `frozen_string_literal: true`? Not sure how that works.